cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -508,10 +550,101 @@ protected void setBootstrapServers(String bootstrapServers) {
         this.bootstrapServers = bootstrapServers;
     }
 
+
+    protected void setRack(Rack rack) {
+        this.rack = rack;
+    }
+
+    protected void setInitImage(String initImage) {
+        this.initImage = initImage;
+    }
```

Comment:
Do we need the setters here? I think we normally tend to just set the values without the use of setters when the setter is not called from outside which does not seem to be the case.

Reply:
Setters removed